package.json

```diff
@@ -5,17 +5,14 @@
   "license": "Apache-2.0",
   "author": "Google Inc.",
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=10"
   },
   "repository": "googleapis/nodejs-storage",
   "main": "./build/src/index.js",
   "types": "./build/src/index.d.ts",
   "files": [
     "build/src",
-    "!build/src/**/*.map",
-    "AUTHORS",
-    "CONTRIBUTORS",
-    "COPYING"
+    "!build/src/**/*.map"
```

Comment:
I've been wanting to start shipping source-maps again, as it can help significantly with debugging (I've been finding it very useful with the `--enable-source-map` flag occasionally.

Reply:
+1 to that.  It's @soldair you have to convince!